features/git-town-append/on-parent-branch.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Appending a branch to a parent branch
```

Comment:
@kevgo and more thoughts here? If this is removed, the on-child-branch can be renamed to on feature branch to match the prepend tests.